examples/grpc/build.gradle

```diff
@@ -10,7 +10,6 @@ dependencies {
     runtime 'org.slf4j:slf4j-simple'
 
     testCompile 'jakarta.annotation:jakarta.annotation-api'
-    testCompile 'junit:junit'
```

Comment:
I think we should add JUnit 5 dependencies instead of this such as `org.junit.jupiter` and `org.junit.platform`. Please see https://github.com/line/armeria/blob/master/build.gradle#L118
Also could you run the script in `https://github.com/line/armeria-examples/blob/master/update-examples.sh` so that everything working as expected? Please, be sure to change [this](https://github.com/line/armeria-examples/blob/master/update-examples.sh#L8) and [this](https://github.com/line/armeria-examples/blob/master/update-examples.sh#L93) before running the script. 😄

Reply:
I will modify `update-examples.sh` of `armeria-example` and come back again 🚀

Reply:
`update-examples.sh` is not part of this repository and @heowc added other commits since the last review. @minwoox, could you take another look?

Reply:
I add `jupiter-api` dependency from each example and sync master.

Reply:
... And I cloned `armeria-examples` to modify `update-examples.sh`. However, there is an exception to the operation and we are checking it.

Reply:
> update-examples.sh is not part of this repository

I agree. 😄 I just wanted to make sure that we do not miss any dependencies by running the script.

Reply:
> I just wanted to make sure that we do not miss any dependencies by running the script.

I agree with this. In fact, I have a lot of changes and issues because I wrote in Kotlin.

I may be lacking or missing something, but if it's written in the legacy application style in the Kotlin build script, I can't use the `springBoot` scope. This is a problem on [this line](https://github.com/line/armeria-examples/blob/master/update-examples.sh#L114), so you have to find another way or problem.

Reply:
Ah yes, we can modify the script later. 😄